### PR TITLE
Update LICENSE.template with CC-BY + BSD from IPR policy update

### PR DESCRIPTION
**Pull request by foolip**
Matches https://github.com/whatwg/sg/pull/115.

### CODE REVIEW

File: LICENSE.template [hunk 1]
```diff
@@ -1,7 +1,10 @@
-Copyright © 2018 WHATWG (Apple, Google, Mozilla, Microsoft).
-This work is licensed under a Creative Commons Attribution 4.0 International License:
+Copyright © 2020 WHATWG (Apple, Google, Mozilla, Microsoft).
+This work is licensed under a Creative Commons Attribution 4.0 International License.
+Alternatively, this work is licensed under a BSD 3-Clause License for incorporation into software.
 
-Attribution 4.0 International
+=======================================================================
```
Comment (annevk) [inline LICENSE.template:5]: Can we follow the precedent set by https://github.com/unicode-org/icu4x/pull/68/files for separate, i.e., use `- - - -` and not denote headers separately?

cc @hsivonen

Comment (foolip) [reply LICENSE.template]: Done.

**Review by annevk — APPROVED** [review]
I have updated this to be aligned with the latest LICENSE.

**Comment by foolip** [comment]
Thanks @annevk!

**Pull request merged**